src/scss/coral-talk-iframe/main.scss

```diff
@@ -408,6 +432,52 @@ body {
 	}
 }
 
+// reply indent
+
+// Unfortunately at the moment it looks like there is a bug in Corals code
+// I believe there should be a .coral-indent class we could use here
+// If this is fixed in the future we can reduce this to a single selector
+.coral-comment .coral-indent-1,
+.coral-comment .coral-indent-2,
+.coral-comment .coral-indent-3,
+.coral-comment .coral-indent-4,
+.coral-comment .coral-indent-5,
+.coral-comment .coral-indent-6 {
+	border-left-style: dotted;
+	border-width: 2px;
+	border-color: #807973;
+	border-image: url(https://www.ft.com/__origami/service/image/v2/images/raw/https%3A%2F%2Fs3-eu-west-1.amazonaws.com%2Fft-next-assets-prod%2Fassets%2Fcomments%2Fborder-dots.svg?source=next&format=svg);
+	border-image-slice: 30%;
+	border-image-repeat: round;
+}
+
+// Replies aren't nested inside each other so each level needs a greater margin
+// There are reply levels from 1 - 6 so we loop over these are produce styles for 2 breakpoints
+// Smaller screens have a smaller indentation than larger screens
+
+@for $i from 1 through 6 {
+	$baseIndent: 20;
```

Comment:
The linter errored here. Should this be? `@for $i from 1 to 6`?

Reply:
Not according to the sass docs and this code is already been used in staging to produce those screenshots 😬 

https://sass-lang.com/documentation/at-rules/control/for

Reply:
I mean I could change it to use `to` and I would need to make it `7` not `6`

Reply:
Here is the output that we are using https://www.ft.com/__origami/service/build/v2/bundles/css?modules=o-comments%40e8474197%3A%2Fsrc%2Fscss%2Fcoral-talk-iframe%2Fmain.scss%2Co-fonts%40%5E4.0.0

Looks like there is a difference between the linting and the build

Reply:
Nope, `through` is valid Sass. hm what's up with the linter?